lhotse/bin/modes/recipes/mini_librispeech.py

```diff
@@ -0,0 +1,27 @@
+import click
+
+from lhotse.bin.modes import recipe
+from lhotse.recipes.mini_librispeech import prepare_mini_librispeech, download_and_untar
+from lhotse.utils import Pathlike
+
+__all__ = ['mini_librispeech']
+
+
+@recipe.command(context_settings=dict(show_default=True))
+@click.argument('corpus_dir', type=click.Path(exists=True, dir_okay=True))
+@click.argument('output_dir', type=click.Path())
+def mini_librispeech(
+        corpus_dir: Pathlike,
+        output_dir: Pathlike
+):
+    """Recipe to prepare the manifests for MiniLibriSpeech task."""
```

Comment:
I don't know how  to resolve this, but what we are calling "recipe" here would be more like "data-prep" in the context of a downstream recipe.  Possibly confusing?

Reply:
I renamed the function `mini_librispeech` to `mini_librispeech_dataprep` in `bin/modes/recipes/mini_librispeech.py`. Is it a solution?

Reply:
OK for me, Piotry can make final decisions.